### PR TITLE
fix: N' attribute and subscript array contents

### PR DESCRIPTION
**Pull request by jirimosinger**
- Fix for [N' attribute issue](https://github.com/eclipse/che-che4z-lsp-for-hlasm/issues/231)
- Subscript array contents are now visible without the need to expand the variable
- Added debugger tests for special behavior of some system variables
- Added general tests for some system variables

### CODE REVIEW

File: parser_library/src/debugging/set_symbol_variable.cpp
Comment (slavek-kucera) [inline parser_library/src/debugging/set_symbol_variable.cpp:72]:
```suggestion
        return get_string_array_value();
```

File: parser_library/src/debugging/set_symbol_variable.h
Comment (slavek-kucera) [inline parser_library/src/debugging/set_symbol_variable.h:45]: given the size, these could be by-value... but I don't mind.

File: parser_library/test/debugging/debugger_test.cpp
Comment (slavek-kucera) [inline parser_library/test/debugging/debugger_test.cpp:165]: 👍

File: parser_library/test/debugging/debugger_test.cpp
Comment (slavek-kucera) [inline parser_library/test/debugging/debugger_test.cpp:290]: should this handle the empty case as well, just for consistency? or at least assert non-emptiness?

File: clients/vscode-hlasmplugin/CHANGELOG.md
Comment (jirimosinger) [reply clients/vscode-hlasmplugin/CHANGELOG.md:17]: Description simplified

File: parser_library/src/context/macro_param_data.cpp
Comment (jirimosinger) [reply parser_library/src/context/macro_param_data.cpp:76]: 👍

File: parser_library/src/debugging/macro_param_variable.cpp
Comment (jirimosinger) [reply parser_library/src/debugging/macro_param_variable.cpp:35]: 👍

File: parser_library/src/debugging/set_symbol_variable.cpp
Comment (jirimosinger) [reply parser_library/src/debugging/set_symbol_variable.cpp:72]: 👍

File: parser_library/src/debugging/set_symbol_variable.h
Comment (jirimosinger) [reply parser_library/src/debugging/set_symbol_variable.h:45]: I don't mind changing it either.

File: parser_library/test/debugging/debugger_test.cpp
Comment (jirimosinger) [reply parser_library/test/debugging/debugger_test.cpp:290]: It sure can.

**Review by slavek-kucera — APPROVED** [review]
we'll need to wait for the release, before we merge.

**Pull request merged**
